### ISSUE TITLE
Unable to run "keystone prisma migrate dev" from Turborepo monorepo

### ISSUE THREAD

**Issue by stuible** [keystonejs/keystone#8858]
# How to reproduce the bug
### 1. I created this repo with Turbo Repo's starter project by running:

```sh
npx create-turbo -e basic
```

### 2. I then added Keystone CMS to `~/apps/keystone` using

```sh
cd apps
npm init keystone-app@latest keystone
```

### 3. I then modified the keystone package.json to have migrate script:

```json
"migrate": "NODE_ENV=development keystone prisma migrate dev"
```

### 4. Move to keystone directory and run keystone command
```sh
cd apps/keystone # move to keystone directory
npm run migrate # run our new migrate npm command
```

### 5. Receive the following output:
```
Error: Prisma Migrate has detected that the environment is non-interactive, which is not supported.
```

# Expected Result
I expect the migrations to be created through an interactive terminal

# Context
- Node v18.18.0
- [Turborepo](https://turbo.build/)

# Repro Repo
https://github.com/stuible/keystone-interactive-cli-bug

Here is a MVP repo that reproduces the issue described above.

**Comment by austinbravodev** [comment]
I'm not using Turbo and the `keystone prisma migrate dev` command fails with the same error when run as an NPM script or via `npx`.

**Comment by flippinjoe** [comment]
This is actually a bug in turborepo.  They swallow stdin (used for automatic migrations in keystone) https://github.com/vercel/turbo/issues/1235

**Comment by austinbravodev** [comment]
Could be both - I think the Keystone CLI creates a non-interactive environment in which the Prisma command runs.

I'm just running it in my terminal, which should be interactive.

This would also explain why `npx prisma migrate dev` works.

**Comment by stuible** [comment]
> This is actually a bug in turborepo. They swallow stdin (used for automatic migrations in keystone) [vercel/turbo#1235](https://github.com/vercel/turbo/issues/1235)

I have my doubts that this is a Turborepo issue because in the reproduction steps above I am bypassing the use of Turborepo completely (as far as I know).

**Comment by SagnikPradhan** [comment]
Weird, `yarn keystone prisma migrate dev` freezes but `yarn dlx prisma@4.x migrate dev` worked flawlessly

**Comment by stuible** [comment]
> Weird, `yarn keystone prisma migrate dev` freezes but `yarn dlx prisma@4.x migrate dev` worked flawlessly

Exactly my experience as well (with the equivalent  npx commands)

**Comment by dcousens** [comment]
Fixed in #8898, will be published soon
